### ISSUE TITLE
Spontaneous concurrent test failure: Do not manipulate content db schema in tests

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#5252]
## Summary

During some of the unit tests, we are deleting the content db.
## System information

_Please specify the KA Lite version you were using and your operating system!_
- Operating system: Circle CI
- Version: 0.16+
## Traceback or relevant snippet from server.log

```
[ERROR] [2016-08-05 17:45:19,082] kalite: No content database file found
    Then the last in-progress video/exercise should be shown # kalite/distributed/features/steps/content_recommendation.py:18 0.029s
      Traceback (most recent call last):
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1173, in run
          match.run(runner.context)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1589, in run
          self.func(context, *args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_recommendation.py", line 20, in impl
          assert get_content_item(content_id=context.videos[1].get("id")).get("path") in context.browser.current_url, "Last in progress video not in %s" % context.browser.current_url
        File "/home/ubuntu/ka-lite/kalite/topic_tools/content_models.py", line 128, in wrapper
          output = function(*args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/topic_tools/content_models.py", line 201, in get_content_item
          return model_to_dict(value)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/playhouse/shortcuts.py", line 110, in model_to_dict
          rel_obj = getattr(model, field.name)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 1138, in __get__
          return self.get_object_or_id(instance)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 1129, in get_object_or_id
          obj = self.rel_model.get(self.field.to_field == rel_id)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 4300, in get
          return sq.get()
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 2792, in get
          return clone.execute().next()
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 2839, in execute
          self._qr = ResultWrapper(model_class, self._execute(), query_meta)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 2535, in _execute
          return self.database.execute_sql(sql, params, self.require_commit)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 3339, in execute_sql
          self.commit()
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 3185, in __exit__
          reraise(new_type, new_type(*exc_args), traceback)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 3331, in execute_sql
          cursor.execute(sql, params or ())
      OperationalError: no such table: item
```
## How to reproduce

This happens only randomly
## Screenshots

https://circleci.com/gh/learningequality/ka-lite/2519

**Comment by benjaoming** [comment]
Fixed in #5268

**Comment by benjaoming** [comment]
Reopening because of similar occurrence:

```

    Then the last in-progress video/exercise should be shown # kalite/distributed/features/steps/content_recommendation.py:18 0.029s
      Traceback (most recent call last):
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1173, in run
          match.run(runner.context)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1589, in run
          self.func(context, *args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_recommendation.py", line 20, in impl
          assert get_content_item(content_id=context.videos[1].get("id")).get("path") in context.browser.current_url, "Last in progress video not in %s" % context.browser.current_url
        File "/home/ubuntu/ka-lite/kalite/topic_tools/content_models.py", line 128, in wrapper
          output = function(*args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/topic_tools/content_models.py", line 201, in get_content_item
          return model_to_dict(value)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/playhouse/shortcuts.py", line 110, in model_to_dict
          rel_obj = getattr(model, field.name)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 1138, in __get__
          return self.get_object_or_id(instance)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 1129, in get_object_or_id
          obj = self.rel_model.get(self.field.to_field == rel_id)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 4300, in get
          return sq.get()
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 2792, in get
          return clone.execute().next()
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 2839, in execute
          self._qr = ResultWrapper(model_class, self._execute(), query_meta)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 2535, in _execute
          return self.database.execute_sql(sql, params, self.require_commit)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 3339, in execute_sql
          self.commit()
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 3185, in __exit__
          reraise(new_type, new_type(*exc_args), traceback)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/peewee.py", line 3331, in execute_sql
          cursor.execute(sql, params or ())
      OperationalError: no such table: item
```

**Comment by benjaoming** [comment]
A bunch of fixes landed during #5350 and I haven't seen this for a while, so closing.
